code/datums/craft/recipes/weapon.dm

```diff
@@ -68,7 +68,7 @@
 	)
 
 /datum/craft_recipe/weapon/throwing_knife
-	name = "throwing knife"
+	name = "throwing knives"
 	result = /obj/item/stack/thrown/throwing_knife
 	steps = list(
 		list(CRAFT_MATERIAL, 2, MATERIAL_PLASTEEL),
```

Comment:
```suggestion
		list(CRAFT_MATERIAL, 3, MATERIAL_PLASTEEL),
```

Reply:
Should be three to craft since you can split the stack, recycle it, and net 1 plasteel.